lib/ret_web/controllers/api/v1/media_controller.ex

```diff
@@ -74,6 +74,14 @@ defmodule RetWeb.Api.V1.MediaController do
   end
 
   defp resolve_and_render(conn, url, version, quality \\ nil) do
+    query = query_for(conn, url, version, quality)
+    value = Cachex.fetch(:media_urls, query)
```

Comment:
You could pipeline value through here (would need to change `render_resolved_media_or_error`)... Not sure if it would actually be clearer since `value` is not being transformed anyway...

This definitely reads more clearly than what we had before.

Also, we didn't have it before but might be worth adding a note that `Cachex.fetch` is what actually kicks off the resolution if its not in the cache.

Reply:
RE: adding a note about `Cachex.fetch`. 

I could be persuaded into adding a note (and I did not know this behavior before now either), but since it's used in several places in the client I am also OK with having people find out about this by consulting the [docs](https://hexdocs.pm/cachex/Cachex.html#fetch/4). I don't feel strongly about this though so I'll add it if you think it's helpful

```elixir
./lib/ret/slack_client.ex:78:      case Cachex.fetch(
./lib/ret/slack_client.ex:93:      case Cachex.fetch(
./lib/ret/slack_client.ex:105:      case Cachex.fetch(
./lib/ret/app_config.ex:49:        Cachex.fetch(:app_config, "")
./lib/ret/app_config.ex:117:    case Cachex.fetch(:app_config_value, key) do
./lib/ret/app_config.ex:126:    case Cachex.fetch(:app_config_owned_file_uri, key) do
./lib/ret/discord_client.ex:66:      case Cachex.fetch(:discord_api, "/guilds/#{community_id}/members/#{provider_account_id}") do
./lib/ret/discord_client.ex:71:      case Cachex.fetch(:discord_api, "/users/#{provider_account_id}") do
./lib/ret/discord_client.ex:80:    case Cachex.fetch(:discord_api, "/users/#{provider_account_id}") do
./lib/ret/discord_client.ex:138:      case Cachex.fetch(:discord_api, "/guilds/#{community_id}") do
./lib/ret/discord_client.ex:146:        case Cachex.fetch(:discord_api, "/guilds/#{community_id}/roles") do
./lib/ret/discord_client.ex:172:        case Cachex.fetch(:discord_api, "/channels/#{channel_id}") do
./lib/ret/discord_client.ex:212:      case Cachex.fetch(:discord_api, "/guilds/#{community_id}/members/#{discord_user_id}") do
./lib/ret/coturn.ex:21:    {_, coturn_secret} = Cachex.fetch(:coturn_secret, :coturn_secret)
./lib/ret_web/controllers/api/v1/media_search_controller.ex:86:    case Cachex.fetch(cache_for_query(query), query) do
./lib/ret_web/controllers/api/v1/media_controller.ex:78:    value = Cachex.fetch(:media_urls, query)
```